src/middlewared/middlewared/plugins/sysdataset.py

```diff
@@ -317,9 +282,8 @@ async def setup(self, mount, exclude_pool):
                 {'properties': {'acltype': {'value': 'off'}}},
             )
 
-        if mount:
-
-            await self.__mount(config['pool'], config['uuid'])
+        if True:
```

Comment:
This looks a little odd.

Reply:
@anodos325 this is to make review diff more clear, I'll dedent this after approval